content/en/tracing/service_catalog/guides/integrating-with-ci-visibility.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Integrating with CI Visibility
+kind: guide
+further_reading:
+- link: "/tracing/service_catalog/"
+  tag: "Documentation"
+  text: "Datadog Service Catalog"
+---
+
+In the Software Delivery lens you can see data from your pipeline executions associated to your services. 
+
+By default, the pipelines are associated to your service by matching the repository URLs with the ones in [CI Visibility][4]. 
+However, you might want more control into what pipelines match with your services. You can configure the pipelines associated 
+with each service by editing the `ci-pipeline-fingerprints` field in your [service metadata][5].
+
+To add a pipeline through the UI, click on `Edit Metadata` in your service and go to Software Delivery. There you are be able to 
+select the pipelines you want. You can also add the pipeline fingerprint directly to the service metadata metadata. You can find
+the pipeline fingerprint for one pipeline by clicking on a pipeline in the pipelines page:
+
```

Comment:
To add or remove a pipeline associated to your service, first click on `Edit Metadata` in your service page and go to _Software Delivery_.

[ add screenshot from demo env of Software Delivery section ]

There are two ways to add or remove an associated pipeline:

1. You can search for and select the pipelines you want to associate.
[ add screenshot from demo env of the search bar and dropdown ]
2. You can add the pipeline fingerprint directly to the service metadata. You can find a pipeline's fingerprint by clicking on a pipeline in the _Pipelines_ [hyperlink] page:
[ screenshot ]